src/common/util/parse-aspect-ratio.js

```diff
@@ -0,0 +1,24 @@
+export default function parseAspectRatio(input) {
+  // Handle 16x9, 16:9, 1.78x1, 1.78:1, 1.78
+  // Ignore everything else
+  function parseOrThrow(number) {
+    const parsed = parseFloat(number);
+    if (isNaN(parsed)) throw new Error(`${number} is not a number`);
+    return parsed;
+  }
+  try {
+    if (input) {
+      const arr = input.replace(':', 'x').split('x');

```

Comment:
Why replace before split? Why not just split on `:` ?

Reply:
This seemed the easiest way to normalize the delimiter before the split. I guess I could pass a regex to the split to support both ":" and "x" as delimiters. I didn't really think about that.